lib/exceptions.json

```diff
@@ -1,2 +1,12 @@
 {
+ "webrtc": [
+    {
+      "rule": "headers.copyright.no-match"
```

Comment:
Too bad, I was hoping specberus could still warn us on other copyright related stuff (e.g. year in copyright, correct links, etc).
Time permitting, I might look at making the exception system more granular, but in the meantime, I've updated the PR